packages/@lwc/engine-core/src/framework/profiler.ts

```diff
@@ -79,10 +84,12 @@ const operationTooltipMapping = [
     'component disconnectedCallback()',
     // errorCallback
     'component errorCallback()',
-    // lwc-hydrate
+    // lwc-render
     'component first rendered',
-    // lwc-rehydrate
+    // lwc-rerender
     'component re-rendered',
+    // lwc-ssr-hydrate
+    'component hydrated from server-rendered HTML',
```

Comment:
`lwc-render` is unfortunately a bit easy to confuse with the existing `render` operation (i.e. the `render()` function itself), but hopefully the `lwc-*` bit helps to disambiguate them. Also the tooltips and colors are different.

Reply:
We could rename it to something like `lwc-root-render` or `lwc-lifecycle-render`. Or change the `render` to something like `render-func`.